frontend/website/static/js/leaderboard.js

```diff
@@ -60,5 +79,44 @@ function start() {
     );
 }
 
+function startChallenges() {
+  gapi.client
+    .init({
+      apiKey: "AIzaSyDIFwMr7SPGCLl_o6e4UZKi1q9l8snkUZs"
```

Comment:
Can you maybe put this API key into a variable at the top of the file and replace both uses?